static/game_operations.js

```diff
@@ -0,0 +1,23 @@
+$("document").ready(newGame);
+
+function update() {
+  $.getJSON( "/update", function( json ) {
+    $( "#pile0" ).html( json.state[0] );
+    $( "#pile1" ).html( json.state[1] );
+    $( "#pile2" ).html( json.state[2] );
+    $( "#pile3" ).html( json.state[3] );
+    $( "#pile4" ).html( json.state[4] );
+    console.log( "JSON Data: " + json.state );
+  });
+};
+
+function newGame() {
+  $.getJSON( "/new", function( json ) {
+    $( "#pile0" ).html( json.state[0] );
+    $( "#pile1" ).html( json.state[1] );
+    $( "#pile2" ).html( json.state[2] );
+    $( "#pile3" ).html( json.state[3] );
+    $( "#pile4" ).html( json.state[4] );
+    console.log( "JSON Data: " + json.state );
```

Comment:
Lots of duplicated code. No way to factorize?

Reply:
May be able to `for loop` it... I wanted to keep it very simple for now.

Reply:
Not just that. The whole function is duplicated.

Reply:
I see. Yeah, there must be a better way. I'll fix it.